TripCheckins/Trip list/UserDefaults+LocalItemsStorage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UserDefaults+LocalItemsStorage.swift
+//  TripCheckins
+//
+//  Created by Nataliya  on 8/27/17.
+//  Copyright © 2017 Nataliya Patsovska. All rights reserved.
+//
+
+import Foundation
+
+let tripsKey = "trips"
+
+extension UserDefaults: LocalItemsStorage {
```

Comment:
I think it's possible to make the below functions more generic (using generics).

Reply:
these are actually redundant, they are not defined by `LocalItemsStorage` 🙈